### PR TITLE
Fixes for sino_360_to_180

### PR DESCRIPTION
**Pull request by dmpelt**
This PR includes some fixes for `sino_360_to_180`, specifically:

* Fix output for `rotation='right'`
* Use both arrays for filling the overlapping region instead of only one array. This avoids using values at the edges of the detector, which may have the most artifacts in practice.
* Simplified the code a bit, using only a single `if` statement
* Renamed `sino_360_t0_180` to `sino_360_to_180` (`sino_360_t0_180` is kept for backward compatibility)
* Changed `rotation is` to `rotation ==`, using an equality comparison instead of an identity comparison (which often gives the correct results due to string caching, but can fail)
* Added a small test for the method

### CODE REVIEW

**Comment by dgursoy** [comment]
Still WIP?

**Comment by dmpelt** [comment]
We have some experimental data that we want to test it on first (we had some errors with it, so I put it back to WIP). I will check whether it works now on Monday.

**Comment by dgursoy** [comment]
sounds good!

**Comment by dmpelt** [comment]
Tests on experimental data did not show any problems, so I think this can be merged!

Based on the tests, I added a note to the docs that the last projections is discarded for odd numbers of angles.

File: test/test_misc/test_morph.py [hunk 1]
```diff
@@ -68,6 +68,14 @@ def test_downsample():
 def test_upsample():
     loop_dim(upsample, read_file('obj.npy'))
 
+def test_sino_360_to_180():
+    test_im = np.random.random((32, 32, 128)).astype(np.float32)
```
Comment (tacaswell) [inline test/test_misc/test_morph.py]: maybe add a test that exercises the odd number of frames behavior?

**Pull request merged**